crates/ruff/src/rules/flake8_pyi/rules/bad_version_info_comparison.rs

```diff
@@ -1,21 +1,24 @@
+use rustpython_parser::ast::{Cmpop, Expr};
```

Comment:
is there something other than `cargo fmt` i have to do here? i have "run rustfmt on save" on in pycharm but the precommit also didn't seem to have caught that

Reply:
This is PyCharm's "Optimize Imports" action. It's kind of a bad habit for me to run + commit it since it's not actually enforced in any automated way, but I often do run it when editing a new file 🙈

Reply:
We used to enforce these via `rustfmt` but it depended on nightly Rust so we removed it.